.travis.yml

```diff
@@ -5,4 +5,4 @@ env:
 node_js:
   - '5.4'
 script:
-  - npm run lint
+  - build/script.bash
```

Comment:
We can also put a **Windows users** section in `README.md`, explaining what changes windows users would need to make.